parsing/parser.mly

```diff
@@ -1717,7 +1718,7 @@ class_expr:
   | let_bindings(no_ext) IN class_expr
       { class_of_let_bindings ~loc:$sloc $1 $3 }
   | LET OPEN override_flag attributes mkrhs(mod_longident) IN class_expr
-      { let loc = ($startpos($2), $endpos($4)) in
+      { let loc = ($startpos($2), $endpos($5)) in
```

Comment:
Could you maybe give a name to the identifier, to avoid using the more fragile `$5`? When we initially ported the grammar we sticked to the yacc convention to minimize the change, but now we should feel free to use names instead of numeric indices, as we evolve the code gradually.